### ISSUE TITLE
img render

### ISSUE THREAD

**Issue by remusgls** [antoniolucasnobar/html-to-rtf-browser#12]
perfect working library, well done, one question though, how do you use the img tag, as I've tried multiple approaches with it, but nothing is rendering:
```
<img style="width: 80px; height: 40px"
       src="data:image/png;base64,....> (with or without </img> tag)
```
I see that the code gets transformed in some way, in the output looks like this:
`....rest of rtf ...  {\pict\pngblip\ 89504e470d0a1a0a0000000d49484....`

Not sure if this is ok, or not, but it doesnt seem to render, in ms word / rtf reader etc./  I either get a empty block, or the base64 string back.

An example of how to use it, would be really amazing,
Thank you!

**Comment by antoniolucasnobar** [comment]
Sorry for so late reply. I missed your message entirely.
Did you manage to solve your issue? 
If not, I will look in my examples to see what is necessary to render an image.

**Comment by antoniolucasnobar** [comment]
hello @remusgls ,
I added [img.html](https://github.com/antoniolucasnobar/html-to-rtf-browser/blob/master/examples/img.html) and the [rtf result](https://github.com/antoniolucasnobar/html-to-rtf-browser/blob/master/examples/img.rtf)

You need to build the project first to generate `dist/main.js` file.

I tried some jpg files and it did not work. I tried this [PNG](https://lh3.googleusercontent.com/proxy/POkHoOMJVqMn6jvVLb0FsnenWXXpURmACe_ywOvccRs_Egy8_-P_P6H_WwtJvle6cXN52e5LffMT16FwQ_ds0ZGJj-QHSjSDWKy-vGNQZJs4BgYPnM8) with transparency and converted at this site: https://base64.guru/converter/encode/image

Tell me if this solves your problem.

I do not have MS Word to test it. I use libreoffice. Maybe you should try it.

And, one more time, sorry the late reply.

**Comment by antoniolucasnobar** [comment]
I looked at https://github.com/MinePlay96/html-to-rtf/commit/5d4fecff03a511d01393a06dccd0f1c28977b433 (where I got the img support), and it looks like that only PNG base64 is supported. Please check the images you are using.

**Comment by antoniolucasnobar** [comment]
I updated the code to support JPG images and BMP (this one is experimental).
I updated the img.html example to show other images.

Let me know if it is still not working to you. Maybe you should provide your example, so we can check and see what is wrong.

**Comment by antoniolucasnobar** [comment]
I released the 1.6.0 in npmjs with JPG support. I am really sorry that I missed your question.

I will close this for now. If you still have problems, comment here or open a new issue. I will try my best to help you in your conversions.

Best Regards,
A. Lucas
